distribution/src/main/resources/docker-compose/docker-compose.yml

```diff
@@ -299,7 +299,7 @@ services:
         condition: service_healthy
     environment:
       JAVA_TOOL_OPTIONS: -agentlib:jdwp=transport=dt_socket,address=*:5007,server=y,suspend=n
-      LOGGING_LEVEL_ORG_ALFRESCO: INFO
+      LOGGING_LEVEL_ORG_ALFRESCO: DEBUG
```

Comment:
Since this is used primarily for dev work then it seems useful to set it to DEBUG so we can easily view the events received and sent by the live ingester.